manager/integration/scripts/upgrade-longhorn.sh

```diff
@@ -9,6 +11,8 @@ CUSTOM_LONGHORN_SHARE_MANAGER_IMAGE="${6}"
 CUSTOM_LONGHORN_BACKING_IMAGE_MANAGER_IMAGE="${7}"
 
 LONGHORN_REPO_DIR="/tmp/longhorn"
+mkdir -p "${LONGHORN_REPO_DIR}"
+rm -rf "${LONGHORN_REPO_DIR}/"
```

Comment:
What is the purpose of `mkdir` then `rm` immediately?

Reply:
remove existing things in /tmp/longhorn since upgrade-longhorn.sh will be executed more than once.

Reply:
Got it. Thanks for clarifying. Suggest putting a comment here.

Reply:
btw, wouldn't this also remove the subdirectory `longhorn`? why not use `rm -rf "${LONGHORN_REPO_DIR}/*"`?

Reply:
Yes if `longhorn` folder not removed, `git clone` will complain `fatal: destination path 'tmp/longhorn' already exists and is not an empty directory.`, so probably the correct way is to remove the `mkdir` part and let `git clone` to create the path for us, and we just need to ensure the existing things are all removed.

I've removed the `mkdir` part and still kept the `rm -rf` part.